script/DeployLocal.s.sol

```diff
@@ -30,12 +30,8 @@ contract DeployLocal is
     DeployPermit2FeeCalculator,
     DeployTransferFromFeeCalculator
 {
-    // 3rd party address
-    address public constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
-    address public constant AAVE_V3_PROVIDER = 0x2f39d218133AFaB8F2B819B1066c7E434Ad94E9e;
```

Comment:
Is there any problem with constant presentations?

Reply:
No issues came up, but when adding deploy files, I noticed that the style of Local would differ from Arbitrum and Polygon, and I wanted to make all the deploy files consistent. Additionally, these two addresses have struct key names to differentiate the meaning of the addresses.